testdata/test-suite-data.json

```diff
@@ -498,7 +498,7 @@
     "type": "huggingface",
     "namespace": "EleutherAI",
     "name": "gpt-neo-1.3B",
-    "version": "797174552ae47f449ab70b684cabcb6603e5e85e",
+    "version": "797174552AE47F449AB70B684CABCB6603E5E85E",
```

Comment:
For consistency with other test data in this file, wouldn't it make more sense to lower-case the SHA1 in the purl in line 496 instead?

Reply:
Ok, probably not that important as you're changing it back in a later commit anyway.

Reply:
Yes I changed it back because that was a mistake as a huggingface version must be lowercase https://github.com/package-url/purl-spec/blob/master/PURL-TYPES.rst#huggingface